api/api-referential/referential-external-client/src/main/java/fr/gouv/vitamui/referential/external/client/OntologyExternalRestClient.java

```diff
@@ -90,4 +92,11 @@ public ResponseEntity<Resource> export(ExternalHttpContext context) {
         final HttpEntity<OntologyDto> request = new HttpEntity<>(null, buildHeaders(context));
         return restTemplate.exchange(uriBuilder.toUriString(), HttpMethod.GET, request, Resource.class);
     }
+
+    public List<ExternalOntologyDto> getInternalOntologyList(ExternalHttpContext context) {
```

Comment:
si y a pas de InternalOntologyDto, tu prefere garder quand meme le renommage ?

Reply:
changed to VitamUiOntologyDto